Journal of Aerospace Information Systems/Blockchain Properties for Near-Planetary, Interplanetary and Metaplanetary Space Domains/Blockchain Properties for Near-Planetary, Interplanetary and Metaplanetary Space Domains.tex

```diff
@@ -256,26 +256,26 @@ \section{Desired Blockchain Properties}\label{sec:desiredproperties}
 
-PoA protocols may be conveniently separated into those that guarantee immediate finality of created blocks, those that guarantee eventual finality and those that do not guarantee finality. A block is said to be final only if it has been added to the blockchain of an honest node and both its position and content may not be changed under any future circumstance (e.g. a network fork, or a rebalancing of the blockchain at a later time). A blockchain consensus protocol provides immediate finality only if any block is final as soon as it is added to the blockchain provided that there are no more faulty nodes than the maximum threshold allowed by the protocol. In other words, immediate finality guarantees that the blockchain cannot fork. A blockchain consensus protocol guarantees eventual finality if blocks become final only after they have been on the chain for ``sufficiently long'' time. In the case that blocks added to the chain never become final, such as in the Clique PoA protocol\cite{Szilagyi-Clique}, then the consensus protocol is said not to guarantee finality.
+PoA protocols may be conveniently separated into those that guarantee immediate finality of created blocks, those that guarantee eventual finality and those that do not guarantee finality. A block is said to be final only if it has been added to the blockchain of an honest node and both its position and content may not be changed under any future circumstance (e.g. a network fork, or a rebalancing of the blockchain at a later time). A blockchain consensus protocol provides immediate finality only if any block is final as soon as it is added to the blockchain provided that there are no more faulty nodes than the maximum threshold allowed by the protocol. In other words, immediate finality guarantees that the blockchain cannot fork. A blockchain consensus protocol guarantees eventual finality if blocks become final only after they have been on the chain for a ``sufficiently long'' time. In the case that blocks added to the chain never become final, such as in the Clique PoA protocol\cite{Szilagyi-Clique}, then the consensus protocol is said not to guarantee finality.
 
-Another property to be considered when choosing PoA protocols, and consensus protocols more in general, is whether they are tolerant to nodes that 
+Another property to be considered when choosing PoA protocols, and consensus protocols more generally, is whether they are tolerant to nodes that 
  act maliciously, 
 are faulty, or suffer communications failures. 
- Such nodes are commonly called Byzantine\cite{lamport1982byzantine}. The ability of a network to correctly operate in the presence of Byzantine nodes is called Byzantine fault tolerance (BFT).
+ Such nodes are commonly called Byzantine\cite{lamport1982byzantine}. The ability of a network to correctly operate in the presence of Byzantine nodes is called Byzantine fault tolerant (BFT).
```

Comment:
I believe "tolerance" is correct in this particular case, because the subject of the sentence is an _ability_. The -ance suffix is used for a "quality or state", c.f. https://www.merriam-webster.com/dictionary/-ance

Do you agree?